packages/apps/kadena-docs/config/rig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rig-package/rig.schema.json",
+
+  "rigPackageName": "@kadena-dev/heft-rig"
```

Comment:
Maybe we can use @rushstack/heft-web-rig/profiles/web
See also https://app.asana.com/0/1202676524474548/1204337512939263